sdk/node/makedoc.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+# -------------------------------------------------------------
```

Comment:
Please align the copyright with the template [here](https://github.com/hyperledger/fabric/blob/master/docs/dev-setup/headers.txt).